flycheck.el

```diff
@@ -7888,6 +7889,48 @@ dirname is test or else default."
   :predicate flycheck-buffer-saved-p
   :working-directory flycheck-rebar3-project-root)
 
+(flycheck-def-config-file-var flycheck-ruumbarc eruby-ruumba ".ruumba.yml"
+  :safe #'stringp)
+
+(flycheck-def-option-var flycheck-ruumba-lint-only nil eruby-ruumba
+  "Whether to only report code issues in Ruumba.
+
+When non-nil, only report code issues in Ruumba, via `--lint'.
+Otherwise report style issues as well."
+  :safe #'booleanp
+  :type 'boolean
+  :package-version '(flycheck . "0.16"))
+
+(flycheck-define-checker eruby-ruumba
+  "An eRuby syntax and style checker using the Ruumba tool.
+
+You need at least Ruumba 0.1.7 for this syntax checker.
+
+See URL `https://github.com/ericqweinstein/ruumba'."
+  :command ("ruumba"
+            "--display-cop-names"
+            "--force-exclusion"
+            "--format" "emacs"
+            "--cache" "false"
+            (config-file "--config" flycheck-ruumbarc)
+            (option-flag "--lint" flycheck-ruumba-lint-only)
+            ;; Ruumba takes the original file name as argument when reading
+            ;; from standard input
+            "--stdin" source-original)
+  :standard-input t
+  :working-directory flycheck-ruby--find-project-root
+  :error-patterns
+  ((info line-start (file-name) ":" line ":" column ": C: "
+         (optional (id (one-or-more (not (any ":")))) ": ") (message) line-end)
+   (warning line-start (file-name) ":" line ":" column ": W: "
+            (optional (id (one-or-more (not (any ":")))) ": ") (message)
+            line-end)
+   (error line-start (file-name) ":" line ":" column ": " (or "E" "F") ": "
+          (optional (id (one-or-more (not (any ":")))) ": ") (message)
+          line-end))
+  :modes (html-erb-mode rhtml-mode)
+  :next-checkers ((warning . eruby-erubis)))
```

Comment:
So you want to `erubis` to run after `ruumba`, but since you added `eruby-ruumba` after `eruby-erubis` in the `flycheck-checkers` list, `erubis` will remain the default checker in ERB and RHTML buffers.  Is this intended?

Reply:
Hmm, my intention was to not change any defaults, just to add ruumba after `erubis`. I have reversed the order, adding next-checker to `eruby-erubis` instead.